From: Paul Gevers <elbrus@debian.org>
Date: Sun, 14 Apr 2024 16:36:00 +0100
Subject: Building documentation with Sphinx uses intersphinx by

Forwarded: not-needed

default, which downloads data from the internet. This isn't allowed
during Debian binary builds.
===================================================================
---
 docs/source/conf.py | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/docs/source/conf.py b/docs/source/conf.py
index adb6f6c..17eaa1d 100644
--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -320,9 +320,11 @@ texinfo_documents = [
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
-    "python": ("https://docs.python.org/3.6", None),
-    "jinja": ("http://jinja.pocoo.org/docs", None),
-    "nbformat": ("https://nbformat.readthedocs.io/en/latest", None),
+# Disabled in Debian because this fetches data during build which isn't allowed.
+# See https://bugs.debian.org/1023040
+#    "python": ("https://docs.python.org/3.6", None),
+#    "jinja": ("http://jinja.pocoo.org/docs", None),
+#    "nbformat": ("https://nbformat.readthedocs.io/en/latest", None),
 }
 
 
